specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -2443,6 +2620,262 @@
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of heating or cooling degree day information"
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "The amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Snow."
+        },
+        "snowDepth": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Snow depth."
+        }
+      }
+    },
+    "DailyHistoricalRecordsResponse": {
+      "type": "object",
+      "properties": {
+        "historicalRecords": {
+          "type": "array",
+          "description": "Historical records for each requested day.",
+          "items": {
+            "$ref": "#/definitions/DailyHistoricalRecords"
+          }
+        }
+      }
+    },
+    "DailyHistoricalRecords": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitYearMaxMinAvg",
+          "description": "Temperature value."
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnitYearMax",
+          "description": "Maximum amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
+          "$ref": "#/definitions/WeatherUnitYearMax",
+          "description": "Maximum snow."
+        }
+      }
+    },
+    "DailyHistoricalNormalsResponse": {
+      "type": "object",
+      "properties": {
+        "historicalNormals": {
+          "type": "array",
+          "description": "Historical normals for each requested day.",
+          "items": {
+            "$ref": "#/definitions/DailyHistoricalNormals"
+          }
+        }
+      }
+    },
+    "DailyHistoricalNormals": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
```

Comment:
Please change the description for degreeDaySummary to:

Summary of heating or cooling degree day information. Degree days are measures of how cold or warm a location is. A degree day compares the mean (the average of the high and low) outdoor temperatures recorded for a location to a standard temperature of 65 degrees F/ 18 degree C.